packages/@react-aria/overlays/src/useUnderlay.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/**
+ * Used when there is an underlay component with the overlay, like a Dialog.
+ */
+export function useUnderlay() {
+  let events;
+  let onPointerDown = (e) => {
```

Comment:
Why is onPointerDown outside of the PointerEvent check?
Do we need to also preventDefault on click or press?

Reply:
Ah don't worry about this PR, it's likely going to be closed as I think #1469 is the way we want to solve this issue
also #1470 will close for the same reason, just waiting on that one to be merged and then I'll handle them